scipy/stats/_continuous_distns.py

```diff
@@ -2923,6 +2923,70 @@ def fit(self, data, *args, **kwds):
 gamma = gamma_gen(a=0.0, name='gamma')
 
 
+class gamma_gompertz_gen(rv_continuous):
+    r""" A Gamma/Gompertz continuous random variable
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `gamma_gompertz` is:
+
+    .. math::
+
+        f\left(x,c,\beta\right) = c*\exp\left(x\right)*\frac{\beta^c}{((\beta-1+\exp\left(x\right))^{c+1})}
```

Comment:
Please clean up the TeX formula: `f(x, c, \beta) = c \beta^c e^x / (e^x + \beta - 1)^{c+1}` (or, if you want to use `\frac`, it's also OK, but please minimize visual clutter.